alf/optimizers/adamw.py

```diff
@@ -162,3 +163,17 @@ def step(self, closure=None):
                 eps=group['eps'])
 
         return loss
+
+
+# NOTE(breakds): This is to make the torch.optim._functional.adamw() call
+# compatible with both pytorch 1.9- and 1.10+. In 1.10 new required parameters
+# are introduced in this function, and the following wrapper supplies the
+# default value for such parameters.
+if "maximize" in inspect.signature(torch.optim._functional.adamw).parameters:
+
+    def _adamw(*args, **kwargs):
+        return torch.optim._functional.adamw(*args, **kwargs, maximize=False)
```

Comment:
``maximize`` is not a required argument of adamw in newer version of pytorch?

Reply:
It actually becomes a required argument in the newer version.

Reply:
It is still optional in pytorch 1.13?
https://pytorch.org/docs/1.13/generated/torch.optim.AdamW.html?highlight=adamw